src/serviceLauncher.ts

```diff
@@ -37,6 +37,8 @@ function main() {
         } else if (service.type == 'ceramic') {
             subprocess = exec(
                 `node node_modules/@ceramicnetwork/cli/bin/ceramic daemon \
+                    --debug \
```

Comment:
Might as well go all the way and use `--verbose` which is the most verbose log level